src/lib/Preview.js

```diff
@@ -1094,6 +1095,23 @@ class Preview extends EventEmitter {
         }
     }
 
+    /**
+     * Handle metrics emitted by the viewer
+     *
+     * @private
+     * @param {Object} [data] - Viewer metric data
+     * @return {void}
+     */
+    handleViewerMetrics(data) {
+        const formattedEvent = {
+            event_name: data.event,
+            ...data.data,
```

Comment:
All loggable events require a `value` property. `value` pertains to additional info about the event. In this case, you can assign `data.data` to the value prop.

ie)
```
const formattedEvent = {
  event_name: data.event,
  value: data.data,
  ...this.createLogEvent()
}
```